src/Illuminate/Http/Concerns/InteractsWithInput.php

```diff
@@ -175,7 +175,7 @@ public function cookie($key = null, $default = null)
     /**
      * Get an array of all of the files on the request.
      *
-     * @return array
+     * @return array<int, \Illuminate\Http\UploadedFile|\Illuminate\Http\UploadedFile[]>
```

Comment:
The key here appears to be wrong, on FormRequest the key will be the name of the file input.

Reply:
Addressed here: https://github.com/laravel/framework/pull/55287

Reply:
Thanks!

Reply:
I was actually looking in to adding types to this a few weeks ago, but backed off when I saw some of the nested logic so I'm glad you handled most of it :)